app/lib/common/utils/utils.dart

```diff
@@ -148,3 +148,14 @@ String formatBytes(int bytes) {
     return '${size.toStringAsFixed(2)} ${suffixes[i]}';
   }
 }
+
+extension ColorUtils on Color {
```

Comment:
can we have a minimal unittest for this? Just adding a bunch of colors and ensuring they come out with the correct value should be fine...